### ISSUE TITLE
DCA: test deployment with updated REST API and docker containers

### ISSUE THREAD

**Issue by milen-sage** [Sage-Bionetworks/data_curator#401]
Once changes from #399, #400,  https://github.com/Sage-Bionetworks/schematic/issues/899 and https://github.com/Sage-Bionetworks/schematic/issues/900,  are incorporated in DCA and schematic respectively, test the deployment in our EC2 development environment.

AC: docker container(s), reducing persistency (i.e. no use of local file system in DCA), and schematic's REST API updates are confirmed working.

**Comment by afwillia** [comment]
[This issue](https://github.com/Sage-Bionetworks/schematic/issues/863) is relevant for dashboard functionality

**Comment by afwillia** [comment]
Successfully have the schematic REST API running from a container and DCA running on a container on the same EC2 instance. DCA can access all of the validation and submission endpoints. The dashboard functionality will be implemented after some updates to schematic.

**Comment by afwillia** [comment]
Also, DCA is set configured with envrionmental variables, not config files. Though the workflow is still dependent on some files on the server.

**Comment by milen-sage** [comment]
@afwillia it sounds like at this point we have #400 resolved and also the app works with the REST API. Once #399 is done, are we at a point where we'll no longer need config files and manifests stored locally, so that we can pass for testing in the Fargate architecture (at least for core features, perhaps not the dashboard functionality)? Or are there other outstanding items?

**Comment by afwillia** [comment]
@milen-sage we need to figure out what to do with `www/config.yml`. Maybe we can create it during the docker image build.

**Comment by milen-sage** [comment]
Yes, since we have @rrchai script that auto generates www/config.yml, we should be able to do that now?

**Comment by afwillia** [comment]
Yeah. I want to wait until the JSON updates for manifest/validate and manifest/submit are both merged into the schematic develop branch. Then once I confirm the DCA non-dashboard code works with that, I'll add the config.yml generation to the dockerfile. I think this can be done next sprint.

**Comment by afwillia** [comment]
All of the relevant schematic endpoints have been updated to accept/return JSON in the develop branch. The DCA + schematic images have been rebuilt.
